testing/testing_view.rst

```diff
@@ -0,0 +1,161 @@
+Testing a view
+==============
+
+Now let's test a view.
+
+Create a new view
+-----------------
+
+Create a new view:
+
+.. code-block:: console
+
+  $ cd plonetraining.testing
+  $ plonecli add view
+
+With this command, a new view will be automatically registered in our package and we don't need to to think about it.
+
+Test the view
+-------------
+
+Like for dexterity types, also for views ``plonecli`` created some basic tests (in ``test_view_training_view.py`` file).
+
+
+Inspecting this file, we could see some interesting things:
+
+.. code-block:: python
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
+        api.content.create(self.portal, 'Folder', 'other-folder')
+        api.content.create(self.portal, 'Document', 'front-page')
+
+    def test_training_view_is_registered(self):
+        view = getMultiAdapter(
```

Comment:
I'd prefer to use plone.api in test as much as possible: 
```
api.content.get_view('training-view', self.portal['other-folder'], self.request)
```

Reply:
me too, but this code is generated by plonecli and i'd like to use it as base for my training..i could use plone.api in my examples and let plonecli generated code as is? And when i have time i promise to fix it on plonecli

Reply:
we can update the code in bobtemplates.plone to use the plone.api call instead, PR welcome.

Reply:
will do it during the sprint ;)